### DISCUSSION TITLE
1.18.2 Compatibility Release

### DISCUSSION THREAD

**Discussion by ghost** [GriefPrevention/GriefPrevention#1848] [Q&A]
If I understand correctly, there is a big difference between 1.18.1 and 1.18.2.  I wanted to put this on a 1.18.2 server, but my host (STIPE) shows that it is only compatible with 1.18.1. 

Is it or when will it be compatible with 1.18.2?

Thanks for your work.

**Comment by bigpresh** [comment]
I'm using GP with 1.18.2 on my server, and it's working fine.

That's a build from commit 0e3a2e97 - the stable GP 16.18 release is from b37afcb which is more recent than the build I'm running (note to self - update to 16.18!)

So - yes - the most recent GP build should work fine with 1.18.2.

**Reply by ghost** [reply]
Thanks heaps.
